css/components/Navigation/navigation.css

```diff
@@ -17,3 +27,101 @@
   vertical-align: middle;
   background: var(--COLOR-RED-500);
 }
+
+.navbar {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem 2rem;
+}
+
+.navbar-align {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+}
+
+.navbar-btn-log {
```

Comment:
you wanted it to be logo?

Reply:
Actually, no first I name it `navbar-btn-login`, but then thought the same class will we use for `log out` as well. That's why, instead of using `login` in the name, I used `log`.